ethereumj-core/src/main/java/org/ethereum/solidity/compiler/CompilationResult.java

```diff
@@ -42,6 +48,52 @@ public static CompilationResult parse(String rawJson) throws IOException {
         }
     }
 
+    @JsonIgnore public Path getContractPath() {
+        if (contracts.size() > 1) {
+            throw new UnsupportedOperationException("Source contains more than 1 contact. Please specify the contract name. Available keys (" + getContractKeys() + ").");
+        } else {
+            String key = contracts.keySet().iterator().next();
+            return Paths.get(key.substring(0, key.lastIndexOf(':')));
+        }
+    }
+
+    @JsonIgnore public String getContractName() {
+        if (contracts.size() > 1) {
+            throw new UnsupportedOperationException("Source contains more than 1 contact. Please specify the contract name. Available keys (" + getContractKeys() + ").");
+        } else {
+            String key = contracts.keySet().iterator().next();
+            return key.substring(key.lastIndexOf(':') + 1);
+        }
+    }
+
+    @JsonIgnore public ContractMetadata getContract(String contractName) {
```

Comment:
Could you add javadoc to getters, especially to this one. There should be mentioned, that if there are more than 1 contract with the same name, only first will be returned, so you should use `{@link getContract(Path contractPath, String contractName)}` for this cases.